css-fonts/styles.css

```diff
@@ -0,0 +1,5 @@
+body {
+  font-family: 'Silkscreen', cursive;
+  font-family: 'DM Sans', sans-serif;
```

Comment:
This second font-family is overriding the first one, try moving it to its own CSS rule set.